meta-xt-prod-devel-rcar-control/recipes-guests/domd/domd.bbappend

```diff
@@ -25,4 +28,9 @@ do_install:append() {
     # Call domd-set-root script
     echo "[Service]" >> ${D}${systemd_unitdir}/system/domd.service
     echo "ExecStartPre=${libdir}/xen/bin/domd-set-root" >> ${D}${systemd_unitdir}/system/domd.service
+
+    if ${@bb.utils.contains('DISTRO_FEATURES', 'enable_virtio', 'true', 'false', d)}; then
+        echo "" >> ${CFG_FILE}
+        echo "driver_domain = 1" >> ${CFG_FILE}
```

Comment:
Is this a some new option for xl.cfg? Looks like it is temporary, as I don't believe, that any upstreamable xen tools code will have option like this